hal/src/main/native/athena/PDP.cpp

```diff
@@ -31,7 +31,7 @@ static constexpr int32_t StatusEnergy = 0x5D;
 
 static constexpr int32_t Control1 = 0x70;
 
-static constexpr int32_t TimeoutMs = 50;
+static constexpr int32_t TimeoutMs = 255;
```

Comment:
This seems like a big increase.  Can it really ever go this high?

Reply:
I changed it to 100ms.